### DISCUSSION TITLE
LZ RULE v4.0.6

### DISCUSSION THREAD

**Discussion by larsonzh** [larsonzh/amdwprprsct#17] [Announcements]
**Asuswrt-Merlin dual WAN port router policy routing service configuration tool**

*华硕改版固件路由器外网双线路接入策略路由服务配置工具*

v4.0.6 (2023/8/7)
1. 重大更新：
   新增 Web 操作页面，直接嵌入 Asuswrt-Merlin 原生界面，风格保持一致，支持梅林 384.5 及以上的固件版本。
   页面位于「外部网络(WAN) - 策略路由」，程序启动时自动挂载，全程提供内容详尽的悬浮式帮助。
   如未出现「 策略路由」页面，说明所用固件不支持该功能。
2. 新增卸载脚本。执行后仅保留与用户有关的配置数据，如不需要，可手工删除。
3. 升级数据抽取、筛选和验证相关的处理算法，减少中间环节，提高前后台交互的数据传输效率和可靠性，减少资源占用，代码执行更为高效和安全。
4. 全面优化代码，修复bug。
5. 更新运营商网段数据文件。

<hr /><em>This discussion was created from the release <a href='https://github.com/larsonzh/amdwprprsct/releases/tag/v4.0.6'>LZ RULE v4.0.6</a>.</em>

**Comment by kingofotaku** [comment]
没想到大佬真的实现了webui，确实挺意外的。宠粉了属于是（

**Reply by larsonzh** [reply]
这东西刚开始折腾时确实费劲，好在迅速找到方法和运气好，没被啥问题卡住
